oneflow/python/test/modules/test_negative.py

```diff
@@ -13,48 +13,90 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-import unittest
+from collections import OrderedDict
 
+import unittest
 import numpy as np
+
 import oneflow.experimental as flow
+from test_util import GenArgList
+
+
+def _test_negtive(test_case, device):
+    input = flow.Tensor(
+        np.array([1.0, -1.0, 2.3]).astype(np.float32),
+        dtype=flow.float32,
+        device=flow.device(device),
+    )
+    of_out = flow.negative(input)
+    np_out = -(input.numpy())
+    test_case.assertTrue(np.array_equal(of_out.numpy(), np_out))
+
+
+def _test_negative_neg(test_case, device):
+    input = flow.Tensor(
+        np.array([1.0, -1.0, 2.3]).astype(np.float32),
+        dtype=flow.float32,
+        device=flow.device(device),
+    )
+    of_out = flow.neg(input)
+    np_out = -(input.numpy())
+    test_case.assertTrue(np.array_equal(of_out.numpy(), np_out))
+
+
+def _test_tensor_negative(test_case, device):
+    input = flow.Tensor(
+        np.array([1.0, -1.0, 2.3]).astype(np.float32),
+        dtype=flow.float32,
+        device=flow.device(device),
+    )
+    of_out = input.negative()
+    np_out = -(input.numpy())
+    test_case.assertTrue(np.array_equal(of_out.numpy(), np_out))
+
+
+def _test_self_tensor_negative(test_case, device):
+    input = flow.Tensor(
+        np.array([1.0, -1.0, 2.3]).astype(np.float32),
+        dtype=flow.float32,
+        device=flow.device(device),
+    )
+    of_out = -input
+    np_out = -(input.numpy())
+    test_case.assertTrue(np.array_equal(of_out.numpy(), np_out))
+
+
+def _test_negative_backward(test_case, device):
+    input = flow.Tensor(
+        np.array([1.0, -1.0, 2.3]).astype(np.float32),
+        dtype=flow.float32,
+        device=flow.device(device),
+        requires_grad=True,
+    )
+    of_out = flow.negative(input)
+    of_out = of_out.sum()
+    of_out.backward()
+    np_grad = [-1.0, -1.0, -1.0]
+    test_case.assertTrue(np.allclose(input.grad.numpy(), np_grad, rtol=1e-05))

```

Comment:
我的理解， `atol` 比 `rtol` 要更重要些，allcose 判断为 True 的公式是：
```
absolute(a - b) <= (atol + rtol * absolute(b))
```
也就是说 `rtol` 是受 `b` 的影响的。
```suggestion
    test_case.assertTrue(np.allclose(input.grad.numpy(), np_grad, atol=1e-05, rtol=1e-05))
```

如果其它地方也存在只设置了 `rtol` 的情况，建议都添加上 `atol`

Reply:
好的。